C/primitive/elements/primitive.c

```diff
@@ -188,54 +209,126 @@ static int32_t either(jetName* result, jetName a, jetName b, bitstream* stream)
  *               NULL != stream
```

Comment:
Update the comment above replacing `prefix of '10'` with `prefix of '1'` since we are now parsing that second bit.